### DISCUSSION TITLE
Multiple harvester question

### DISCUSSION THREAD

**Discussion by mauver** [Chia-Network/chia-blockchain#2072] [Q&A]
I have a main machine and other machine for harvester. 

When I connect to the farm peer on harvester machine, i can see the my harvester on main machine's "Your harvester Network"  menu. 

So this machine is farming now? no need to connect the plot directory by network drive? 

My main machine has the 5 plots and second machine has the 2 plots, but I can see the only 5 plots, but It's okay?

**Comment by kenneth-1024** [comment]
I have the same confusion with you.
